patterns/ecs-v1/redis

```diff
@@ -1,3 +1,3 @@
 REDISTIMESTAMP %{MONTHDAY} %{MONTH} %{TIME}
-REDISLOG \[%{POSINT:pid}\] %{REDISTIMESTAMP:timestamp} \* 
-REDISMONLOG %{NUMBER:timestamp} \[%{INT:database} %{IP:client}:%{NUMBER:port}\] "%{WORD:command}"\s?%{GREEDYDATA:params}
+REDISLOG \[%{POSINT:[process][pid]:int}\] %{REDISTIMESTAMP:timestamp} \*
+REDISMONLOG %{NUMBER:timestamp} \[%{INT:[redis][database][id]} %{IP:[client][ip]}:%{POSINT:[client][port]:int}\] "%{WORD:[redis][command][name]}"\s?%{GREEDYDATA:[redis][command][args]}
```

Comment:
hey @webmat and @ebeahan we'll be using some custom fields for the redis monitoring command captures:
`"redis" => { "command" => { "name" => "rpush", "args" => "\"my:key\" \"{\\\"data\\\":\"cdr\\\"}\""  } }`

Reply:
The ECS fields I see are mapped correctly 👍 

Yes I agree there's no general place to capture queries to storage systems yet, in ECS. This is coming, but there's nothing concrete I can point you to yet. So I agree with using custom fields 👍 

It's worth pointing out that Beats custom fields for the Redis slowlog are slightly different than those you have here. Since these aren't in ECS, I don't mind if you adapt to those or not, but I'm sharing for awareness: https://github.com/elastic/beats/blob/master/filebeat/module/redis/slowlog/_meta/fields.yml

My sense is that we will get general fields to capture queries in ECS before the Stack 8.0 release, so I'll let you decide how you prefer to handle this.

Reply:
Thanks, did no actually see those as I wasn't really looking at slow-log - assumed this to be a "monitor" log.
There's definitely differences on how LS parses almost everything compared to Beats.
Which is unfortunate and a bit out-of-scope here, anyhow please keep pointing these out anyways :wink: 
This would deserve a `REDISSLOWLOG` pattern but given that Beat's slowlog parsing also feels outdated, I am likely going to keep these as is (reluctant on adding the `redis.slowlog` prefix - since the pattern is named differently).